<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_icon">
       <property name="text">
        <string>icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_filetitle">
       <property name="text">
        <string>filetitle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_parent_dirpath">
     <property name="text">
      <string>parent dirpath</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_page_count">
         <property name="text">
          <string>page count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>é¡µ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_filesize">
       <property name="text">
        <string>filesize</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_preview">
     <property name="text">
      <string>preview</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QToolButton" name="toolButton_open_path">
       <property name="text">
        <string>open path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_refresh">
       <property name="text">
        <string>refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_delete">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
